packages/message-relayer/src/exec/run.ts

```diff
@@ -82,6 +88,16 @@ const main = async () => {
     l2BlockOffset: L2_BLOCK_OFFSET,
     l1StartOffset: L1_START_OFFSET,
     getLogsInterval: GET_LOGS_INTERVAL,
+    logger: USE_SENTRY
+      ? new Logger({
+          name: 'Message_Relayer',
+          sentryOptions: {
+            release: `message-relayer@${process.env.npm_package_version}`,
+            dsn: SENTRY_DSN,
+            environment: ETH_NETWORK_NAME,
+          },
+        })
+      : undefined,
```

Comment:
If I understand correctly, by passing in `undefined` here, it will hit the constructor code in `BaseService` such that a regular logger will be created? Without knowing the implementation of `BaseService`, its hard to know that a logger will be used at all so perhaps its better to just pass through a logger without `sentryOptions` if `USE_SENTRY` is `true`